libass/ass_cache_template.h

```diff
@@ -90,7 +90,7 @@ START(glyph, glyph_hash_key)
     GENERIC(int, italic)
     GENERIC(unsigned, scale_x) // 16.16
     GENERIC(unsigned, scale_y) // 16.16
-    FTVECTOR(outline) // border width, 16.16
+    FTVECTOR(outline) // border width, 26.6
```

Comment:
Due to using _d6 instead of _d16? I guess I'm not sure why this needs a precision change, but it's somehow mandated due to border_scale changing the value range? We could also just avoid this "risk" and use a separate key.